src/modules/projects/components/project-details-content/project-details-content.tsx

```diff
@@ -0,0 +1,19 @@
+import { Spin } from "antd";
+import { useProjectQuery } from "../../api";
+import { ProjectDetailsForm } from "../project-details-form";
+
+interface ProjectDetailsFormProps {
+    projectId: string;
+}
+
+const ProjectDetailsContent = ({ projectId }: ProjectDetailsFormProps) => {
+    const { project, loading } = useProjectQuery({ projectId });
+
+    if (!project && loading) return <Spin size="large" />;
```

Comment:
should we really check !project for loading?

Reply:
removed